ballet/update.py

```diff
@@ -0,0 +1,61 @@
+import git
+import os
+import tempfile
+import yaml
+
+from cookiecutter.main import cookiecutter
+
+from ballet.compat import pathlib
+from ballet.quickstart import generate_project
+
+YML_FILE = 'ballet.yml'
+
+def _find_ballet_dir(path):
```

Comment:
The unstated assumption is that the `ballet-update` command can be run from any subdirectory of the project root, where the project root contains `ballet.yml` file. Unfortunately we detect `ballet.yml` files from multiple locations currently, see `ballet.project.get_config_paths`. It is okay to make a stricter assumption here... check if the current directory contains `ballet.yml` file, if not exist with error.